servicetalk-grpc-netty/src/test/java/io/servicetalk/grpc/netty/ProtocolCompatibilityTest.java

```diff
@@ -480,42 +565,54 @@ private static void testBlockingRequestResponse(final BlockingCompatClient clien
     }
 
     private static void testRequestResponse(final CompatClient client, final TestServerContext server,
-                                            final boolean streaming) throws Exception {
+                                            final boolean streaming,
+                                            @Nullable final String compression) throws Exception {
```

Comment:
While you are here, please remove `throws Exception`, it's never thrown from this test.